Makefile

```diff
@@ -1,13 +1,15 @@
 BINARY=gmg
 
 build:
-	go build -o ${BINARY}
-
-install:
 	go get github.com/fatih/color
```

Comment:
is it necessary to get the color package every time we compile? I am not super knowledgeable on the efficiency of the `go get` command but that seems wasteful.